frontend/components/validator/charts/CumulativeExtrinsicsChart.vue

```diff
@@ -0,0 +1,585 @@
+<template>
+  <div>
+    <div v-if="loading" class="text-center py-4">
+      <Loading />
+    </div>
+    <div v-else>
+      <b-button-group size="sm" class="mt-5">
```

Comment:
It should be a separate component so you can reuse it.

Reply:
Done